src/update.js

```diff
@@ -44,9 +46,16 @@ async function main() {
     'clustersubscription': { install: argv.cs || argv['clustersubscription'], uri: `${fileSource}/ClusterSubscription/${filePath}` },
     'remoteresource': { install: argv.rr || argv['remoteresource'], uri: `${fileSource}/RemoteResource/${filePath}` },
     'mustachetemplate': { install: argv.mtp || argv['mustachetemplate'], uri: `${fileSource}/MustacheTemplate/${filePath}` },
-    'encryptedresource': { install: argv.er || argv['encryptedresource'], uri: `${fileSource}/EncryptedResource/${filePath}` }
   };
 
+  // Handle deprecated resources
+  for( const resourceName of Object.getOwnPropertyNames( resourcesObj ) ) {
+    if( Object.hasOwn( deprecatedResourcesObj, resourceName ) ) {
+      delete resourcesObj[ resourceName ];
+    }
+  }
```

Comment:
Third copy of the code from above? Same optimization? Move to a common function?